acs-integration-tests/src/test/java/com/ge/predix/integration/test/ACSMeteringIT.java

```diff
@@ -174,6 +191,30 @@ private Double getEntitlementUsageByFeatureId(final String featureId, final Stri
         return entitlement.getCurrentUsedAmount();
     }
 
+    private double getSteadyEntitlementUsage(final String featureId, final String subscriptionId)
+            throws Exception {
+        LOGGER.info("Entered Meter Steady Stage");
+
+        double entitlementUsage_1 = 0.0;
+        double entitlementUsage_2 = 0.0;
+
+        for( int i = 0; i < MAX_STEADY_ITERATIONS; i++ ) {
+            entitlementUsage_1 = getEntitlementUsageByFeatureId(featureId, subscriptionId);
```

Comment:
Here's a nitpick. Consider giving entitlementUsage_1 a more friendly/intelligible name. It's subjective but here are some suggestions: usageBeforeWait, usageAfterWait, usageBeforeSleep, usageAfterSleep, or something similar. The reason it helps is that it will make the subsequent if statement more readable. Ask your self what is more meaningful:
```entitlementUsage_1 == entitlementUsage_2```
or
```usageBeforeSleep == usageAfterSleep```

You could even consider abstracting away the code which test the usage pattern into a method with an intelligible name such as "isUsageSteady" then you could write:

```if (isUsageSteady(featureId, subscriptionId))```

This code is sparse on comments so even a small refactoring like that improves the legibility.